docs/source/using/troubleshooting.adoc

```diff
@@ -13,11 +13,11 @@ toc::[]
 This section contains solutions to common problems that you might
 encounter while using Minishift.
 
-[[troubleshooting-kvm]]
-== KVM driver
+[[troubleshooting-drivers]]
+== Troubleshooting problems with drivers
```

Comment:
I'd argue that one first has general issues and then driver issues. WDYT?

Also "Troubleshooting" seems redundant for this heading and make is imo unnecessarily long.